packages/shared/src/mock/metadata.ts

```diff
@@ -151,10 +151,10 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
         }),
       ],
     }),
-    makeNxGroup('nexus_entry_no_default', 'NXentry', {
+    makeNxGroup('nexus_no_default', 'NXprocess', {
       defaultPath: undefined,
       children: [
-        makeNxGroup('ignore_me', 'NXprocess'),
+        makeNxGroup('ignore_me', 'NXentry'),
```

Comment:
Replacing the class here to check that we look for an `NXdata` child first, regardless of its position.